client/app/templates/project.hbs

```diff
@@ -159,6 +159,39 @@
 
     <Messages::Assistance class="large-margin-top" />
 
+    <h3>
+      City Planning Staff
+    </h3>
+    <ul class="no-bullet">
+      {{#each this.project.teamMembers as |teamMember|}}
+        <li class="grid-x medium-margin-bottom">
+          <div class="cell shrink small-padding-right">
+            <FaIcon @icon='user' @fixedWidth={{true}} @transform='down-1' class="text-gray" />
+          </div>
+          <div class="cell auto">
+            <h5 class="no-margin">{{teamMember.name}}</h5>
+            <p class="text-small tiny-margin-bottom">
+              <i>{{teamMember.role}}</i>
+            </p>
+            <p class="text-small tiny-margin-bottom">
+              <a href="mailto:{{teamMember.email}}">{{teamMember.email}}</a>
+            </p>
+            <p class="text-small tiny-margin-bottom">
+              {{teamMember.phone}}
+            </p>
+          </div>
+        
+          {{#if @canDelete}}
+          <div class="cell shrink medium-padding-left">
+            <button type="button" class="text-red-dim" {{on "click" this.tryRemoveEditor}}>
```

Comment:
I don't think `tryRemoveEditor` is an action on the project.js controller

Reply:
👏  Another great catch. Same for `@ canDelete`

Reply:
Whoaaaa my b